2-multitenant/terraform.tfvars

```diff
@@ -19,5 +19,8 @@ apps = {
     "ip_address_names" : [
       "frontend-ip",
     ]
+    "certificates" : {
+      "frontend" : ["frontend.example.com"]
```

Comment:
should the key be "frontend-example-com" ?

Reply:
Yeah - or alternatively `networking.gke.io/pre-shared-certs` should just be `frontend`.  For now I'll just make them both `frontend-example-com` as eventually we'll need a valid value as this is a placeholder.